repl.js

```diff
@@ -67,6 +67,26 @@ const setup = async () => {
     dllAddress: replServer.context.dll.options.address,
     attributeStoreAddress: replServer.context.attributeStore.options.address
   }, { gas: GAS_LIMIT, gasPrice: GAS_PRICE })
+
+  // parameterizer
+  replServer.context.dll = await deployDll(replServer.context.web3, replServer.context.admin)
+  replServer.context.attributeStore = await deployAttributeStore(replServer.context.web3, replServer.context.admin)
```

Comment:
Do we need to deploy the `dll` and the `attributeStore` multiple times in this block? Not sure what happens if we do, but it looks like they are all written back to the same variable in `replServer.context`.

Reply:
do not deploy multiple times, it wouldn't break the context, but you'd have multiple deployed contracts on the ganache instance